src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -80,11 +80,6 @@ public interface IJsonRpcConfig : IConfig
         DefaultValue = "[Eth,Subscribe,Trace,TxPool,Web3,Personal,Proof,Net,Parity,Health,Rpc]")]
     string[] EnabledModules { get; set; }
 
-    [ConfigItem(
-        Description = "Enable experimental eth_simulate call (see https://github.com/ethereum/execution-apis/pull/484 for API refrence) ",
-        DefaultValue = "false")]
-    public bool EnabledRpcSimulate { get; set; }
-
```

Comment:
Remove from the class too, not only from interface!

Reply:
done